src/index.ts

```diff
@@ -1,104 +1,155 @@
-/* eslint-disable @typescript-eslint/no-explicit-any */
-import axios, { AxiosPromise, AxiosRequestConfig, Method } from 'axios';
-import axiosRetry from 'axios-retry';
-import * as fastq from 'fastq';
+import fastq from "fastq";
+import fetchRetry from "fetch-retry";
+import packageJson from "../package.json" assert { type: "json" };
 
-import { VERSION } from './version';
+const API_URL = "https://api.zenrows.com/v1/";
 
-const API_URL = 'https://api.zenrows.com/v1/';
+type HttpMethods = "GET" | "POST" | "PUT";
 
 interface ClientConfig {
-    concurrency?: number;
-    retries?: number;
+  concurrency?: number;
+  retries?: number;
 }
-export interface ZenRowsConfig {
-	autoparse?: boolean;
-	css_extractor?: string;
-	js_render?: boolean;
-	premium_proxy?: boolean;
-	proxy_country?: string;
-	wait_for?: string;
-	wait?: number;
-	block_resources?: string;
-	window_width?: number;
-	window_height?: number;
-	device?: string;
-	original_status?: boolean;
-	custom_headers?: boolean;
-	[x: string]: unknown;
+export interface ZenrowsConfig {
```

Comment:
Typo? Or is there a reason to not use the proper name (`ZenRows`)?

Reply:
That's a 100% a Typo. Fixed!